TOOLS/umpv

```diff
@@ -26,62 +26,103 @@ Note: you can supply custom mpv path and options with the MPV environment
       is already running), this will be ignored.
 """
 
-import sys
 import os
-import socket
-import errno
-import subprocess
-import string
+import platform
 import shlex
+import string
+import subprocess
+import sys
+from collections.abc import Iterable
 
-files = sys.argv[1:]
+if platform.system() == "Windows":
+    import pywintypes
+    import win32file
+    import winerror
```

Comment:
so these are actually third party modules. I think that makes this somewhat less useful.

Reply:
Really? I think everyone has hundreds of python modules installed anyway, it's just distribution model. But yes, python supports `AF_PIPE` only for multiprocess communication. It's not implemented for example in socket or anything else. Those are thin wrappers over win32 api.

Reply:
But, actually, since we only need to open and write, probably I should be able to use native file io. Stay tuned.

Reply:
Fixed.